camstack/cams/cred1.py

```diff
@@ -474,55 +473,66 @@ class Apapane(CRED1):
 
         'FILTER01': ('UNKNOWN', 'IRCAMs filter state', '%-16s', 'FILTR'),
         'RET-ANG1': (0.0, 'Position angle of first retarder plate (deg)', '%20.2f', 'HWPAG'),
-
+    }
+    # WIP: migrating WCS to a more automated approach.
+    '''
         'CDELT1': (4.5e-6, 'X Scale projected on detector (#/pix)', '%20.8f', 'CDEL1'),
         'CDELT2': (4.5e-6, 'Y Scale projected on detector (#/pix)', '%20.8f', 'CDEL2'),
-        'C2ELT1': (4.5e-6, 'X Scale projected on detector (#/pix)', '%20.8f', 'C2EL1'),
-        'C2ELT2': (4.5e-6, 'Y Scale projected on detector (#/pix)', '%20.8f', 'C2EL2'),
+        'CUNIT1': ('DEGREE    ', 'Units used in both CRVAL1 and CDELT1', '%-10s', 'CUNI1'),
+        'CUNIT2': ('DEGREE    ', 'Units used in both CRVAL2 and CDELT2', '%-10s', 'CUNI2'),
         'CTYPE1': ('RA--TAN   ', 'Pixel coordinate system', '%-10s', 'CTYP1'),
```

Comment:
`RA---TAN` triple dash the RA ctype so it takes up 3 chars (from Wcs pdf I don't want to find the section)